### PR TITLE
ciao-down: Add support for clear containers development

### PR DESCRIPTION
**Pull request by mcastelino**
Add the packages needed to build clear container binaries
from sources

Signed-off-by: Manohar Castelino <manohar.r.castelino@intel.com>

### CODE REVIEW

File: testutil/ciao-down/cloudinit.go [hunk 1]
```diff
@@ -273,7 +273,14 @@ write_files:
...
      printf "Your go code is at {{.GoPath}}\n"
-     printf "You can also edit your code on your host system"
+     printf "You can also edit your code on your host system \n"
+     printf "To build cc-oci-runtime from sources \n"
+     printf "git clone https://github.com/01org/cc-oci-runtime \n"
+     printf "cd cc-oci-runtime \n"
+     printf "./autogen.sh --with-cc-kernel=/usr/share/clear-containers/vmlinux.container --with-cc-image=/usr/share/clear-containers/clear-containers.img \n"
+     printf "make \n"
+     printf "make check \n"
+     printf "make install \n"
```
Comment (markdryan) [inline testutil/ciao-down/cloudinit.go]: sudo make install

**Comment by markdryan** [comment]
make failed for me

```
proxy/protocol.go:23:2: cannot find package "github.com/01org/cc-oci-runtime/proxy/api" in any of:
	/usr/local/go/src/github.com/01org/cc-oci-runtime/proxy/api (from $GOROOT)
	/home/markus/go-fork/src/github.com/01org/cc-oci-runtime/proxy/api (from $GOPATH)
Makefile:5227: recipe for target 'cc-proxy' failed
```

I think we need a go get github.com/01org/cc-oci-runtime/proxy/api somewhere.

File: testutil/ciao-down/cloudinit.go (hunk 1 above)
Comment (markdryan) [inline testutil/ciao-down/cloudinit.go]: Also, do I need to restart docker or some other service to use the newly compiled version?  If so, how?

Comment (mcastelino) [inline testutil/ciao-down/cloudinit.go]: @markdryan you should not need to restart docker or any other service. I left out make install. 
Let me add that.

**Pull request merged**